Posts/2021/02/XML-NewEmployees.md

```diff
@@ -0,0 +1,180 @@
+```yaml
+---
+post_title: 'Leveraging XML with New Employee PowerShell Scripts'
+username: s-r-turner
+categories: ActiveDirectory, XML, PowerShell
+tags: XML, ActiveDirectory
+featured_image:
+summary: using xml referencing to bolster up your new employee scripts with information.
+---
+```
+
+# Leveraging XML With New Employee PowerShell Scripts
+
+I'm going to show how you can leverage XML files for referencing information, to help bolster up your Active Directory user accounts.
```

Comment:
What information and why do you need to olster AD User Accounts. 
Ensure you make the business case for why you are recommending this approach.